### ISSUE TITLE
the NET seems not working well with pretrained weight replknet31_base_224_pt1k_basecls.pkl

### ISSUE THREAD

**Issue by dejavudejavu** [DingXiaoH/RepLKNet-pytorch#11]
I put two dichotomies classification datasets into training,one about construction crack and background and the other about cat and dog,both experiments were conducted with default parameters and replknet31_base_224_pt1k_basecls.pkl.The result shows that the loss didn't decline and the evaluation accurracy stay unchanged, the same as the percentage of one particular class,which means the RepLKNet  had no classification capability after efficient training.
![image](https://user-images.githubusercontent.com/61358424/160038319-d3fab8f5-2eb1-49e5-aef1-3ab64778fdcf.png)
percentage of cat pics is also about 49%

**Comment by DingXiaoH** [comment]
Hi, this seems a BaseCls model, so please raise an issue at the MegEngine version's repo https://github.com/megvii-research/RepLKNet/issues
